README.md

```diff
@@ -64,34 +64,89 @@ data
 ├── metric_datasets
 │   ├── dogs
 │   ├── shoes
+│   ├── celebrities
+│   ├── logos
 ├── models
 │   ├── dogs
 │   └── shoes
+│   └── celebrities
+│   └── logos
 └── qdrant_storage
     ├── aliases
     ├── collections
     │   ├── dogs
     │   └── shoes
+    │   └── celebrities
+    │   └── logos
     └── collections_meta_wal
 interactive
 metrics
 notebooks
 scripts
 
 ```
-This structure is split as follows:
+The structure of the `data` folder is split as follows:
   * `metric_datasets` - split into folders corresponding with data categories, each containing raw pictures that were used for model training and are being pulled as a result of visual search.
   * `models` - split into folders corresponding with data categories, each containing pretrained deep learning models,
   * `qdrant_storage` - storage for vector search engine (Qdrant), each data category has its own collection.
 
-### Local
+### Local - Manual
+
+Installation using the terminal window:
+* Install ***git***, ***docker*** packages.
+* `cd` to your target directory.
+* Clone [repository](https://github.com/stxnext/visual-similarity-search) (preferably use SSH cloning).
+* Download `data.zip` file from the [Google Drive](https://drive.google.com/file/d/1ru2FHcV8xBDqX9Q_zgAkJYDlYWlszmOD/view?usp=sharing) and unpack it to the repository so that the folder structure above is retained.
```

Comment:
Add some information why you need to download that and what it contains. Probably everyone will be asking themself the same question.

Reply:
Done